yarn-project/aztec-nr/aztec/src/context/globals/private_global_variables.nr

```diff
@@ -0,0 +1,14 @@
+use dep::protocol_types::traits::Serialize;
+
+// docs:start:private-global-variables
+struct PrivateGlobalVariables {
+    chain_id: Field,
+    version: Field,
+}
+// docs:end:private-global-variables
+
+impl Serialize<2> for PrivateGlobalVariables {
```

Comment:
Should this be using a constant instead?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4473/commits/b57d8b23ce23c4a3a9c949998f8892af55468acc